jetty-core/jetty-fcgi/jetty-fcgi-server/src/test/java/org/eclipse/jetty/fcgi/server/ExternalFastCGIServerTest.java

```diff
@@ -22,15 +22,15 @@
 import org.eclipse.jetty.client.ContentResponse;
 import org.eclipse.jetty.client.HttpClient;
 import org.eclipse.jetty.fcgi.client.transport.HttpClientTransportOverFCGI;
-import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIfSystemProperty;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 public class ExternalFastCGIServerTest
 {
     @Test
-    @Disabled("Relies on an external server")
+    @EnabledIfSystemProperty(named = "external.fastcgi.port", matches = "[0-9][0-9][0-9][0-9]")
```

Comment:
I don't understand this change?

Should not be enough to tag it?
Even if it is supposed to connect to `localhost:9000`, tagging it external would skip it.

```java
@Tag("external")
@Test
public void testExternalFastCGIServer()
```